common/src/main/resources/assets/goofyplugin/lang/en_us.json

```
@@ -6,6 +6,7 @@
   "goofyplugin.permissions.value.bumpscocity": "bumpscocity",
   "goofyplugin.permissions.value.bumpscocity.tooltip": "goofy:whatDoesBumpscocityDo()",
   "figura.docs.events.error": "(GoofyPlugin) An event that is called when something errors.\nReturn true to keep error, and return false to ignore error",
+  "figura.docs.events.entity_render": "(GoofyPlugin) An event that is called on entity render.\nReturn true to cancel rendering,\nthe first argument is the entity being rendered",
```

return value comes after arguments
```suggestion
  "figura.docs.events.entity_render": "(GoofyPlugin) An event that is called on entity render\nCalled with the entity being rendered\nReturn true to prevent the entity rendering",
```